ps_distributionapiclient.php

```diff
@@ -32,6 +32,12 @@
 
 class Ps_Distributionapiclient extends Module
 {
+    /** @var string */
+    public const SUBMIT_NAME = 'update-configuration';
+
+    /** @var string */
+    public const SEND_SHOP_URLS = 'DISTRIBUTIONAPI_SEND_SHOP_URLS';
```

Comment:
It's nitpicking but can we remove the final S? It's only one URL
```suggestion
    public const SEND_SHOP_URL = 'DISTRIBUTIONAPI_SEND_SHOP_URL';
```